commonknowledge/stopwatch
# Allow other Flourish visualisation types

## Description


Allows other stories, which are the other visualisation type, in Flourish.

Also fixes a boolean field that should be optional rather than required.

## Motivation and Context




Fixes a client request.

## How Can It Be Tested?




1. On a page of any appropriate kind, embed a story (like https://public.flourish.studio/story/1041919/)
2. Publish the page.
3. See this story appear.
4. Add a regular single visualisation (like https://public.flourish.studio/visualisation/7778204/)
5. Publish the page.
4. See this single visualisation appear.

## How Will This Be Deployed?



No special changes required.

## Screenshots (if appropriate):

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:


- My change requires a change to the documentation.
- I've updated the documentation accordingly.
